### DISCUSSION TITLE
keyclu wont launch at login

### DISCUSSION THREAD

**Discussion by ObedH26** [Anze/KeyCluCask#101] [Q&A]
Hi there,
I have a problem, in keyclu setting the launch at login option is on, and even if i add keyclue to the launch at login in the system preference of my mac, after rebooting my mac it won't launch and keyclu is deleted from the list in my system preferences.
Any idea what's up ?

**Comment by Anze** [comment]
Hello [ObedH26](https://github.com/ObedH26),
Thanks for reporting the issue. Could you please let me know which macOS version you're currently running and the way how(where) you installed the app?

**Comment by ObedH26** [comment]
Good evening, looking at my safari history i got it from github one month ago redirected from this specific authors website

i am running macOS Sonoma 14.0 on an M1 pro
Have a good one thanks for reaching me

**Comment by Anze** [comment]
Hello, I've attempted to reproduce the issue, but unfortunately, I couldn't replicate it in VM. My research online indicates that this issue might be related to the operating system itself, and it may require adjusting certain permissions or removing specific files (depending on your specific case). I'm sorry that I couldn't provide a solution, but feel free to ask if you have any other questions or need assistance with something else.

**Comment by ObedH26** [comment]
Hey, thanks for your answer, no problem ill let know here if i find a solution

**Comment by pkzrnvch** [comment]
I have exactly the same problem, also running Sonoma on M1 Pro.
